doc/classes/SkeletonIK.xml

```diff
@@ -26,35 +49,47 @@
 			</description>
 		</method>
 		<method name="stop">
 			<return type="void">
 			</return>
 			<description>
+				Stops applying IK effects on each frame to the [Skeleton] bones and also calls [method Skeleton.clear_bones_global_pose_override] to remove existing overrides on all bones.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="interpolation" type="float" setter="set_interpolation" getter="get_interpolation" default="1.0">
+			Interpolation value how much the IK results are applied to the current skeleton bone chain. A value of [code]1.0[/code] will overwrite all skeleton bone transforms completely while a value of [code]0.0[/code] will visually disable the SkeletonIK. A value at or below [code]0.01[/code] also removes [code]bones_global_pose_override[/code] on [Skeleton].
 		</member>
 		<member name="magnet" type="Vector3" setter="set_magnet_position" getter="get_magnet_position" default="Vector3( 0, 0, 0 )">
+			Secondary target position (first is [code]target[/code] property or [code]target_node[/code]) for the IK chain. Use magnet position (pole target) to control the bending of the IK chain. Only works if the bone chain has more than 2 bones. The middle chain bone position will be [code]linear interpolated[/code] with the magnet position.
 		</member>
 		<member name="max_iterations" type="int" setter="set_max_iterations" getter="get_max_iterations" default="10">
+			Number of iteration loops used by the IK solver to produce more accurate (and elegant) bone chain results.
 		</member>
 		<member name="min_distance" type="float" setter="set_min_distance" getter="get_min_distance" default="0.01">
+			The minimum distance between bone and goal target. If distance is below this value the IK solver stops further iterations.
 		</member>
 		<member name="override_tip_basis" type="bool" setter="set_override_tip_basis" getter="is_override_tip_basis" default="true">
+			If [code]true[/code] overwrites the rotation of the tip bone with the rotation of the [code]target[/code] (node).
 		</member>
 		<member name="root_bone" type="String" setter="set_root_bone" getter="get_root_bone" default="&quot;&quot;">
+			The [String] name of the current root bone, the first bone in the IK chain.
 		</member>
 		<member name="target" type="Transform" setter="set_target_transform" getter="get_target_transform" default="Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0 )">
+			First target of the IK chain where the tip bone is placed and, if [code]override_tip_basis[/code] is [code]true[/code], how the tip bone is rotated. If a [code]target_node[/code] path is available the nodes transform is used instead and this property is ignored.
 		</member>
 		<member name="target_node" type="NodePath" setter="set_target_node" getter="get_target_node" default="NodePath(&quot;&quot;)">
+			Target node [NodePath] for the IK chain. If available the nodes current [Transform] is used instead of the [code]target[/code] property.
```

Comment:
```suggestion
			Target node [NodePath] for the IK chain. If available, the node's current [Transform] is used instead of the [member target] property.
```